Applications/WebPortal/WebApp/library/source/components/country_selection_box.tsx

```diff
@@ -62,9 +61,9 @@ export class CountrySelectionBox extends React.Component<Properties> {
     const content = (() => {
       if(this.props.readonly) {
         return (
-          <TextField readonly
-            value={this.props.countryDatabase.fromCode(this.props.value).name}
-            displaySize={this.props.displaySize}/>);
+          <div style={{...boxSizing, ...CountrySelectionBox.STYLE.readonly}}>
```

Comment:
Use the ```TextField``` here. The solution should not involve reimplementing anything.

Reply:
I can't because I need the one in the profile page and that one is no longer being exported.